database.go

```diff
@@ -158,41 +163,76 @@ type datastore struct {
 var _ writestore = &datastore{}
 
 func (db *datastore) now() string {
-	if db.driverName == driverSQLite {
+	switch db.driverName {
+	case driverSQLite:
 		return "strftime('%Y-%m-%d %H:%M:%S','now')"
+	case driverMySQL:
```

Comment:
For switch statements like this, you should be able to combine cases just to keep things as clean as possible, e.g.

```
case driverMySQL, driverPostgres:
```